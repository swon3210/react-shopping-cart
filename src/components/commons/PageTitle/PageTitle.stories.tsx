import PageTitle, { Props } from './PageTitle';

export default {
  title: 'Components/Commons/PageTitle',
  component: PageTitle,
  argTypes: {},
};

const Template = (args: Props) => <PageTitle {...args} />;

export const Default = Template.bind({});
(Default as any).args = {
  children: '주문목록',
};
