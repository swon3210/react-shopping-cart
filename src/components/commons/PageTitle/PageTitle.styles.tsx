import styled from 'styled-components';

export const PageTitle = styled.div`
  display: flex;
  justify-content: center;
  border-bottom: 4px solid ${({ theme }) => theme.BLACK_500};
`;

export const Title = styled.div`
  color: ${({ theme }) => theme.TEXT_COLOR};
  font-size: 32px;
  font-weight: bold;
  margin-bottom: 29px;
`;
