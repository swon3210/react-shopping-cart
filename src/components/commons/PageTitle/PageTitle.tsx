import * as Styled from './PageTitle.styles';

export interface Props {
  children: React.ReactNode;
}

const PageTitle = ({ children }: Props) => {
  return (
    <Styled.PageTitle>
      <Styled.Title>{children}</Styled.Title>
    </Styled.PageTitle>
  );
};

export default PageTitle;
