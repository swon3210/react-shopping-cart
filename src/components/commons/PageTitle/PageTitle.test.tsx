import PageTitle from './PageTitle';
import { render } from '@testing-library/react';

describe('PageTitle Component', () => {
  it('PageTitle Snapshot', () => {
    const pageTitleUtil = render(<PageTitle>TEST</PageTitle>);
    expect(pageTitleUtil.asFragment()).toMatchSnapshot();
  });
});
