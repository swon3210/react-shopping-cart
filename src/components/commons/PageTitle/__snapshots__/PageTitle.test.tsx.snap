// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageTitle Component PageTitle Snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-bdvvaa guGwOB"
  >
    <div
      class="sc-gsDJrp cLkgbs"
    >
      TEST
    </div>
  </div>
</DocumentFragment>
`;
